packages/shared/src/styles/components/menu.css

```diff
@@ -12,6 +12,7 @@
   }
 
   &.menu-secondary {
+    @apply transform-none;
```

Comment:
It doesn't seem like the right fix. What do you think?
The difference between the primary and secondary variations should be only design not positioning

Reply:
Some background:

I've changed the way the react contexify works, so we don'tr need to calculate it's position.
And that works for context menus.

this search bar however,  takes a class "react-contexify" but is not really one.
Therefor it already had some different rendering.

This transform was introduced by me in the latest changes, and should not affect this search bar, as it only uses the wrapper styling, not the actual contexify positions.

How we load the search dropdown:

```html
<div
      role="menu"
      className="top-full mt-1 react-contexify menu-secondary"
      style={{
        position: 'absolute',
        top: placement?.y,
        left: placement?.x,
        opacity: isOpen ? 1 : 0,
        width: placement?.width,
      }}
    >
```

vs a proper react contexify element:

```html
<Menu
        disableBoundariesCheck
        id={id}
        className="menu-primary"
        animation="fade"
        onHidden={() => setVisibility(false)}
        style={{ width: menuWidth }}
      >
```

Reply:
Thanks for the context makes more sense now but still, I think it's not trivial to just override the transform in the `menu-secondary`.
Let's override it in the style of the search dropdown instead.